### PR TITLE
Add new options to `girder-install plugin``

### PR DESCRIPTION
**Pull request by zachmullen**
* `--skip-web` will skip the web client build step
* `--skip-reqs` will skip installing requirements.txt files

### CODE REVIEW

**Comment by zachmullen** [comment]
@girder/developers any objections to these new options?

File: girder/utility/install.py [hunk 1]
```diff
@@ -175,6 +174,12 @@ def main():
     plugin.add_argument('-s', '--symlink', action='store_true',
                         help='Install by symlinking to the plugin directory.')
 
+    plugin.add_argument('--skip-web', action='store_true',
+                        help='Skip the step of running the web client build.')
+
+    plugin.add_argument('--skip-reqs', action='store_true',
+                        help='Skip the step of pip installing the requirements.txt file.')
+
```
Comment (msmolens) [inline girder/utility/install.py]: Nit: the options could be more self-explanatory if written as `--skip-web-client` and `--skip-requirements`.

**Comment by mgrauer** [comment]
No objections, they seem like a benefit.

Can you update the docstrings in `install_web` and `install_plugin` to include the full set of expected `opts` keys?

install_plugin: (plugin, skip_reqs, development, symlink, force, npm, skip_web)

install_web: (npm, development)

I have not tested this install.

**Comment by zachmullen** [comment]
@mgrauer my plan is to create a branch for 2.0-integration to change from accepting `opts` to accepting a list of kwargs so that this can be used from something besides the command line. I'll push that PR soon.

**Comment by zachmullen** [comment]
Everyone happy with this?

**Pull request merged**